src/test/Makefile

```diff
@@ -524,7 +524,7 @@ ifeq ($(shell $(CC) -v 2>&1 | grep -q "clang version" && echo "clang"),clang)
 # Please revisit versions when new clang version arrive. Supported versions: { Linux / OSX: 7 - 14 }
 # Travis reports CC_VERSION of 4.2.1
 CC_VERSION_MAJOR := $(firstword $(subst ., ,$(CC_VERSION)))
-CC_VERSION_CHECK_MIN := 7
+CC_VERSION_CHECK_MIN := 4
```

Comment:
Please do not touch this setting.

We should rather increase this version to 10 - 12 for linux. Not sure about OSX. But should be done in a separate PR.

Reply:
argh, this where one of the try to find the issue.